lib/parser/current.rb

```diff
@@ -46,7 +46,7 @@ def warn_syntax_deviation(feature, version)
     CurrentRuby = Ruby23
 
   when /^2\.4\./
-    current_version = '2.4.0'
+    current_version = '2.4.3'
```

Comment:
Nope, we still don't even support 2.4.1. If you remove this change I'll merge the rest.

Reply:
Interesting, thanks!